Assets/Scripts/Timeline.cs

```diff
@@ -2105,7 +2105,15 @@ public double GetPercentPlayedFromSeconds(double seconds)
 
 		bool convertWavToOgg(string wavPath, string oggPath) {
 			System.Diagnostics.Process ffmpeg = new System.Diagnostics.Process();
-			string ffmpegPath = Path.Combine(Application.streamingAssetsPath, "FFMPEG", "ffmpeg.exe");
+
+            string ffmpegPath = Path.Combine(Application.streamingAssetsPath, "FFMPEG", "ffmpeg.exe");
```

Comment:
nit: I know we're kinda all over the place with tabs/spaces in the project, but if you could fix up the files to conform to whatever they use that would be awesome.

Reply:
Yeah realised that some files were using tabs, spaces of 3 and 4. Changed my commit so each file uses whatever the file is using.